extensions/wikia/DesignSystem/i18n/build.js

```diff
@@ -37,6 +37,12 @@ var languages = fs.readdirSync(rootDir).filter(function (file) {
 languages.forEach(function (lang) {
 	var i18n = require(rootDir + '/' + lang + '/' + filename);
 
+	// Oasis uses zh for simplified Chinese, while the DS uses zh for traditional
+	// so we need to correct the lang code here so the Oasis fallbacks take effect
+	if (lang === 'zh') {
+		lang = 'zh-hant';
```

Comment:
`zh-hant`. `zh-hans` is simplified Chinese, which is the code used in DS. `zh` in DS is traditional Chinese or `zh-hant` for Oasis.

In Oasis, `zh` should fallback to `zh-hans`, but it had it's own definition so was using traditional Chinese. `zh-tw` fallsback to `zh-hant` then to `zh-hans`, but since `zh-hant` was undefined it was showing `zh-hans` or simplified Chinese. If that all makes sense. :)